File: pset1/cash.c/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float e;
    do
    {
    e = get_float("Change owed: ");
    }
    while(e < 0);
    int cents = round(e * 100);
    int coins = 0;

    while(cents >= 25)
    {
        cents -= 25;
        coins++;
    }
    while(cents >= 10)
    {
        cents -= 10;
        coins++;
    }
    while(cents >= 5)
    {
        cents -= 5;
        coins++;
    }
    while(cents >= 1)
    {
        cents -= 1;
        coins++;
    }
    printf("%i\n", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float
total = 0
while (total <= 0):
    total = get_float("Change owed: ")
total = total * 100
penny = 0
nickel = 0
dime = 0
quarter = 0
while (total > 0):
    if total >= 25:
        total -= 25
        quarter += 1
    elif total >= 10:
        total -= 10
        dime += 1
    elif total >= 5:
        total -= 5
        nickel += 1
    elif total >= 1:
        total -= 1
        penny += 1
print(penny + nickel + dime + quarter)
        
    
